CodeGeneration/Sources/Utils/CodeGenerationFormat.swift

```diff
@@ -66,7 +66,7 @@ public class CodeGenerationFormat: BasicFormat {
 
   public override func visit(_ node: MemberDeclListItemSyntax) -> MemberDeclListItemSyntax {
     let formatted = super.visit(node)
-    if node.indexInParent != 0 && !node.decl.is(EnumCaseDeclSyntax.self) {
+    if !node.decl.is(EnumCaseDeclSyntax.self) {
```

Comment:
This got internal. 
So now we get a newline on each start. 

Not sure this is by design or an bug?